postgresql-containers/build/postgres-postgis/Dockerfile

```diff
@@ -0,0 +1,11 @@
+arg basever
+FROM percona/percona-postgresql-operator:${basever}-postgres
+
+USER 0
+
+COPY build/postgres-postgis/oracle-linux-ol8.repo /etc/yum.repos.d/
+
+RUN set -ex; \
+    curl -Lf -o /etc/pki/rpm-gpg/RPM-GPG-KEY-oracle https://yum.oracle.com/RPM-GPG-KEY-oracle-ol8; \
+    microdnf -y module disable postgresql; \
+    microdnf -y install percona-postgis33 percona-postgis33-client;
```

Comment:
@inelpandzic are you sure that we should not add these lines: 
```
EXPOSE 5432

ADD bin/postgres-gis /opt/crunchy/bin/postgres

ENTRYPOINT ["/opt/crunchy/bin/uid_postgres.sh"]

USER 26

CMD ["/opt/crunchy/bin/start.sh"]
```

Did you test it?

Reply:
I'll recheck it @hors , maybe you're right.